components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1941,6 +1942,9 @@ def full_viewer(request, iid, conn=None, **kwargs):
     @return:            html page of image and metadata
     """
 
+    server_id = request.session['connector'].server_id
+    server_name = Server.get(server_id).server
```

Comment:
Something (I haven't been able to find what) changed in omeroweb which means `Server._registry` is empty, so `Server.get(server_id)` returns None instead of an object.